src/ui/SmartSnippet/SmartSnippetCollapsibleSuggestion.ts

```diff
@@ -124,23 +130,29 @@ export class SmartSnippetCollapsibleSuggestion {
     return container;
   }
 
-  private buildSourceTitle(title: string, clickUri: string) {
-    return this.buildLink(title, clickUri, SOURCE_TITLE_CLASSNAME);
+  private buildSourceTitle() {
+    return this.buildLink(this.source.title, SOURCE_TITLE_CLASSNAME);
   }
 
-  private buildSourceUrl(url: string) {
-    return this.buildLink(url, url, SOURCE_URL_CLASSNAME);
+  private buildSourceUrl() {
+    return this.buildLink(this.source.clickUri, SOURCE_URL_CLASSNAME);
   }
 
-  private buildLink(text: string, href: string, className: string) {
-    const element = $$('a', { className, href }).el as HTMLAnchorElement;
+  private buildLink(text: string, className: string) {
+    const element = $$('a', { className: `CoveoResultLink ${className}` }).el as HTMLAnchorElement;
     element.innerText = text;
+    new ResultLink(element, {}, { ...this.bindings, resultElement: this.collapsibleContainer.el }, this.source);
```

Comment:
Eventually, we might see some request to allow customization of this in the same manner we allow you to customize ResultLink in the standard result list.

Being able to modify the href with `hrefTemplate`, or to "open in new window by default", stuff like that.

For now, what we have here should suffice, but something to keep in mind if we start seeing requests/bug reports that ask for things like "I want to change how results are opened in question answering, like we do for the result list".